themes/custom/lex/scss/main.scss

```diff
@@ -61,3 +61,24 @@ ol, ul, dl {
     }
   }
 }
+
+.lex-region-sidebar {
+  #block-eventsblock.section-padding, #block-meetingsblock.section-padding {
```

Comment:
you could just move .section-padding inside the two IDs.

`#id, #id { .class { } }`